canonical-web-and-design/ubuntu.com
# Refactor search results page

## Done

- Add form factor headers only if user selects form factor from secondary nav, or selects the "All ubuntu certified <form> hardware" link from the overview page. If user selects filters and clicks `update` the header should now be the standard search results header.
- Fix SoC path in `/certification/views.py`
- Fix error by null checking `forms` in `/certification/views.py`
- Add `filters=True` flag to URL to be able to differentiate between form factor view and search results view. 

## QA

- Start here: https://ubuntu-com-9551.demos.haus/certification
- Click through to the search results from the "All ubuntu certified `<form>` hardware" links from the cards on the overview page. This should show the form-factor specific headings. 
- Use the secondary nav to select "Desktop" or "IoT" etc - these should show the specific form-factor headings - then select a filter and click update and you should be back to the generic search results page.
- Go back to the overview page and select from the drop down and type something to do a normal search. The header should be the generic search results header. 
- See the [wireframe](https://docs.google.com/document/d/1vIf1vC1mg72JbzmahrcKCaRZN-qMLCgin_i9iqjUxpM/edit#heading=h.b6q302go5m5l) for a description on the differences between the generic search results page and the form-factor pages.

#### Known issues:
- nav not highlighting correct page - @carkod will work on fixing [this](https://github.com/canonical-web-and-design/certification.ubuntu.com/issues/178) next week. 
- Numbers on overview page are incorrect: [issue](https://github.com/canonical-web-and-design/ubuntu.com/issues/9554) filed here


## Issue / Card

Fixes[ #176](https://github.com/canonical-web-and-design/certification.ubuntu.com/issues/176)
